build.sbt

```diff
@@ -50,6 +50,7 @@ lazy val dockerHashing = project
 
 lazy val cromwellApiClient = project
   .withLibrarySettings("cromwell-api-client", cromwellApiClientDependencies)
+  .dependsOn(engine)
```

Comment:
that doesn't seem right

Reply:
This is an artifact of me trying to use Cromwell's Metadata types to parse the return types.  

I do need a dependency on `cwl` for it's `File` class.